src/types/GradeLevel.ts

```diff
@@ -0,0 +1,13 @@
+enum GradeLevel {
```

Comment:
I was actually thinking of using numbers here but we can fix that in a later PR since this is hidden functionality anyways.

Reply:
so for the middle school option: would it be '6-8'? or simply 8.

Reply:
I was originally thinking 8. Though this is actually a good point to discuss. You could write a one-pager with a proposal for how to represent the various grade levels. I just realized you might have cases of 5th year high schoolers, 5th year undergrad students, etc. that might not fit well with the standard grade level-ing system.